packages/rgbpp/src/rgbpp/utils/transaction.ts

```diff
@@ -1,19 +1,14 @@
 import { BaseCkbVirtualTxResult } from '@rgbpp-sdk/ckb';
-import { BtcAssetsApi } from '@rgbpp-sdk/service';
+import { BtcAssetsApi, BtcAssetsApiError } from '@rgbpp-sdk/service';
 
 export interface RgbppTxGroup {
   ckbVirtualTxResult: BaseCkbVirtualTxResult | string;
   btcTxHex: string;
 }
 
 export interface SentRgbppTxGroup {
-  btc: {
-    txId?: string;
-    error?: Error | unknown;
-  };
-  ckb: {
-    error?: Error | unknown;
-  };
+  btcTxId?: string;
+  error?: string;
```

Comment:
The `string` type error is inconvenient when tracing the error.

Reply:
The python SDK can not parse the TypeScript Error object and  I think the string is enough.
```
[{'error': 'BtcAssetsAPI returned an error: (400) sendrawtransaction RPC error: {"code":-25,"message":"bad-txns-inputs-missingorspent"}'}]
```

Reply:
> The python SDK can not parse the TypeScript Error object and I think the string is enough.

But non-python users can use this method for their node/browser apps as well.

Reply:
I suggest to add `console.error` to try-catch block for debug

Reply:
https://github.com/ckb-cell/rgbpp-sdk/pull/283/commits/1c11426f6c1ca6ab196929b0e6641ccfdbdc4c20